src/types/client/mediawiki/index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module "mediawiki" {
+	export interface ImmediatelyInvokingEntityLoadedHookHandler {
```

Comment:
The lack of this definition causes a error in the other parts (like src/client/init.ts). The only alternative is to use 'any' at MW.hook, which I tried to avoid, because there is some sort of definition for this.